### PR TITLE
fix(antd): fix Cascader in form readPretty mode display error #3250

### PR DESCRIPTION
**Pull request by fengwuxp**
_Before_ submitting a pull request, please make sure the following is done...

- [ ] Ensure the pull request title and commit message follow the [Commit Specific](https://formilyjs.org/guide/contribution#pr-specification) in **English**.
- [ ] Fork the repo and create your branch from `master` or `formily_next`.
- [ ] If you've added code that should be tested, add tests!
- [ ] If you've changed APIs, update the documentation.
- [ ] Ensure the test suite passes (`npm test`).
- [ ] Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

**Please do not delete the above content**

---

see issues [Cascader readPretty 模式下回显错误](https://github.com/alibaba/formily/issues/3250)
[验证 fix demo](https://codesandbox.io/s/dazzling-goldwasser-qlwhfw?file=/App.tsx)

### CODE REVIEW

File: packages/antd/src/preview-text/index.tsx [hunk 1]
```diff
@@ -239,18 +239,43 @@ const Cascader: React.FC<React.PropsWithChildren<CascaderProps<any>>> =
...
+      // unified conversion to multi selection mode
+      return props.multiple ? val : [val]
     }
     const getLabels = () => {
       const selected = getSelected()
-      const labels = getValueByValue(dataSource, selected)
-        ?.filter((item) => isValid(item))
-        ?.map((item) => item?.whole?.join('/'))
-        .join(', ')
+      const values = findSelectedItems(dataSource, selected)
```
Comment (fengwuxp) [inline packages/antd/src/preview-text/index.tsx:273]: 为了避免修改 getValueByValue 影响其他组件，重新写了查找选中值的逻辑。

Comment (janryWang) [reply packages/antd/src/preview-text/index.tsx:273]: fusion组件要不要也改一下？

Comment (fengwuxp) [reply packages/antd/src/preview-text/index.tsx:273]: 好的，晚上处理一下

Comment (fengwuxp) [reply packages/antd/src/preview-text/index.tsx:273]: fusion Cascader readPretty 已修复

测试发现一个新问题：
   fusion 的级联选择组件普通模式回填表单值也有问题,参见： [demo](https://codesandbox.io/s/reverent-wildflower-lz0mhk?file=/App.tsx:3428-3432)
回填查找逻辑好像是使用数组的最后一项，要是级联选择最后一个选项有重复（相对上一个选项）的话会就会有问题（参见例子的租户环境级联选中）

Comment (janryWang) [reply packages/antd/src/preview-text/index.tsx:273]: 那这个是fusion的问题吧，先忽略

**Pull request merged**